includes/TinyMCE.php

```diff
@@ -105,6 +108,19 @@ public function web_stories_tinymce_scripts() {
 		}
 	}
 
+
+	/**
+	 * Print required inline script in the footer.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @return void
+	 */
+	public function print_footer_scripts() {
+		// Can't use wp_localize_script() because there's no script handle.
+		printf( "<script id='web-stories-tinymce-button-js'>\n%s\n</script>\n", 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
+	}
```

Comment:
Why not use [wp_add_inline_script](https://developer.wordpress.org/reference/functions/wp_add_inline_script/) here.

Reply:
As the comment says: Can't use `wp_localize_script()` because there's no script handle.

`wp_add_inline_script` also requires a script handle. TinyMCE plugins don't have script handles.

Reply:
Couldn't we enqueue with an empty string?
```
public function enqueue_assets() {
		$this->enqueue_style( 'wp-components' );

		// Can't use enqueue_script() because the script needs to be loaded via the mce_external_plugins filter.
		$asset = $this->get_asset_metadata( 'tinymce-button' );
		wp_enqueue_script( 'tinymce-button', '', $asset['dependencies'], $asset['version'], true );
                wp_add_inline_script(  'tinymce-button', 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';');
	}
```